### PR TITLE
support list or tuple in DatanodeViewer

### LINKED ISSUE #867: BUG-Deal with lists or tuples in DataNodeViewer
**Description**
Deal with lists or tuples in DataNodeViewer

**How to reproduce**

- A code fragment

``` python
import math

import pandas

from taipy.config import Config, Scope
from taipy.core import clean_all_entities_by_version, create_global_data_node
from taipy.gui import Gui

# This was expected to remove the .taipy directory... and does simply not
try:
    clean_all_entities_by_version()
except:
    pass

dn_cfg2 = Config.configure_data_node(id="my_node2", default_data=[1,2,3], scope=Scope.GLOBAL)

data_node2 = create_global_data_node(dn_cfg2)

page = """
<|{data_node2}|data_node|>
"""

Gui(page=page).run()

```

**Expected behavior**
Be able to display and edit a simple list in datanodeviewer

**Acceptance Criteria**
- [ ] Ensure new code is unit tested, and check code coverage is at least 90%
- [ ] Create related issue in taipy-doc for documentation and Release Notes if relevant

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
and check DatanodeViewer chart local config before applying

resolves #867

### CODE REVIEW

File: frontend/taipy/src/DataNodeChart.tsx [hunk 1]
```diff
@@ -237,19 +237,24 @@ const DataNodeChart = (props: DataNodeChartProps) => {
 
     const [config, setConfig] = useState<ChartConfig | undefined>(undefined);
     useEffect(() => {
+        let localConf;
```
Comment (FabienLelaquais) [inline frontend/taipy/src/DataNodeChart.tsx:240]: No init value? Or relying on the language's default?

**Review by FabienLelaquais — APPROVED** [review]
👌

File: frontend/taipy/src/DataNodeChart.tsx (hunk 1 above)
Comment (FredLL-Avaiga) [reply frontend/taipy/src/DataNodeChart.tsx:240]: > No init value? Or relying on the language's default?

Default is undefined and that's good with me

Comment (FabienLelaquais) [reply frontend/taipy/src/DataNodeChart.tsx:240]: 👌

**Pull request merged**